node/node.go

```diff
@@ -590,6 +602,11 @@ func (node *Node) Start() error {
 						return false
 					}
 
+					// ignore the further processing of the p2p messages as it is not intended for this node
+					if ignore {
```

Comment:
can check the error message here to do the ignoring, rather than using a extra boolean.

Reply:
as above, trying to be explicit and more readable, that's why a separate variable is used.